config/localization/en/app.json

```diff
@@ -1635,6 +1643,7 @@
       "ADJUSTED_IMR": "Adjusted Margin",
       "ADJUSTED_IMR_BODY": "We have temporarily {ACTION} the Initial Margin Requirements on {MARKET} to {PERCENT} due to volatility. {ADDITIONAL_NOTE}",
       "BLOCK_REWARD": "Block Reward",
+      "CANCELING_ALL_ORDERS": "Canceling all orders",
```

Comment:
nit: there's an existing string that says `Cancelling` in this repo - can we update that to `Canceling` just for consistency?